projects/1.20/assets/fungal-infection-spore/spore/lang/zh_cn.json

```
@@ -0,0 +1,422 @@
+
+
+  "item.spore.scent_spawnegg": "毒气",
+  "item.spore.scent_overcharged_spawnegg": "过载毒气",
+  "item.spore.claw_spawnegg": "利爪",
+
+  "entity.spore.knight": "菌骑士",
+  "entity.spore.griefer": "侵袭者",
+  "entity.spore.inf_human": "寄染人类",
+  "entity.spore.inf_husk": "寄染尸壳",
+  "entity.spore.inf_wanderer": "寄染流浪商人",
+  "entity.spore.braiomil": "舔食者",
+  "entity.spore.inf_villager": "寄染村民",
+  "entity.spore.inf_witch": "寄染女巫",
+  "entity.spore.inf_pillager": "寄染劫掠者",
+  "entity.spore.spitter": "喷吐者",
+  "entity.spore.leaper": "激跃者",
+  "entity.spore.slasher": "鞭笞者",
+  "entity.spore.inf_vindicator": "寄染卫道士",
+  "entity.spore.inf_evoker": "寄染唤魔者",
+  "entity.spore.inf_hazmat": "寄染研究员",
+  "entity.spore.howler": "咆哮者",
+  "entity.spore.stalker": "追迹者",
+  "entity.spore.brute": "蛮兽",
+  "entity.spore.scent": "毒气",
+  "entity.spore.claw": "利爪",
+  "entity.spore.busser": "邪灵",
+  "entity.spore.volatile": "巫裂餮",
+  "entity.spore.inf_drowned": "寄染溺尸",
+  "entity.spore.inf_player": "寄染冒险者",
+  "entity.spore.scamper": "窜扰者",
+  "entity.spore.mound": "肉囊",
+  "entity.spore.vigil": "哨戒者",
+  "entity.spore.usurper": "僭主者",
+  "entity.spore.spit": "酸液",
+  "entity.spore.thorn": "棘刺者",
+  "entity.spore.umarmed": "恒守者",
+  "entity.spore.proto": "雏生联合心智",
+  "entity.spore.reconstructor": "菌兽构建者",
+  "entity.spore.wendigo": "林中鹿菌",
+  "entity.spore.inquisitor": "殉道者",
+  "entity.spore.brot": "长舌魔",
+  "entity.spore.sieger": "强袭魔",
+  "entity.spore.gazenbreacher": "凝袭魔",
+  "entity.spore.sieger_tail": "强袭魔之尾",
+  "entity.spore.hindenburg": "菌械空堡",
+  "entity.spore.tumoroid_nuke": "瘤弹",
+  "entity.spore.licker": "舔舐者",
+  "entity.spore.plagued": "零号病人",
+  "entity.spore.lacerator": "突袭者",
+  "entity.spore.wave": "菌潮",
+  "entity.spore.illusion": "幻形魔",
+  "entity.spore.braurei": "酿毒体",
+  "entity.spore.verva": "保育体",
+  "entity.spore.delusioner": "妄念魔",
+  "entity.spore.gastgaber": "菌主",
+  "entity.spore.howitzer": "菌械重炮",
+  "entity.spore.howit_arm": "菌械重炮肢体",
+
+  "enchantment.spore.symbiotic_reconstitution.desc": "加强你和你的装备之间的共生纽带，处在共生状态下将修复其损伤。",
+  "enchantment.spore.cryogenic_aspect.desc": "使被攻击者感到刺骨的寒意，减慢其移动速度并造成冰冻伤害。",
+  "enchantment.spore.gastric_spewage.desc": "使血肉具有胆汁的特性，受害者将如同被胆汁喷淋。",
+  "enchantment.spore.corrosive_potency.desc": "使血肉具有腐蚀的特性，你的攻击将能够融化受害者的防御。",
+  "enchantment.spore.unwavering_nature.desc": "使血肉不再遗忘其寄染生命的原始欲望，即使那是它的主人",
+  "enchantment.spore.mutagenic_reactant.desc": "真菌的不稳定性旨在帮助其同族而不是其宿主。",
+
+
+  "entity.spore.thrown_tumor": "肿瘤",
+  "entity.spore.thrown_spear": "寄染长矛",
+  "entity.spore.acid_ball": "酸液球",
+  "entity.spore.bile": "胆汁",
+  "entity.spore.acid": "酸液",
+
+
+  "enchantment.spore.symbiotic_reconstitution": "共生重塑",
+  "enchantment.spore.cryogenic_aspect": "低温附加",
+  "enchantment.spore.gastric_spewage": "胃液反流",
+  "enchantment.spore.corrosive_potency": "腐蚀之效",
+  "enchantment.spore.unwavering_nature": "恒定自然",
+  "enchantment.spore.mutagenic_reactant": "诱变反应",
+
+
+  "effect.spore.mycelium_ef": "菌丝",

```

```suggestion
  "effect.spore.mycelium_ef": "菌丝感染",
```
Mycelium Infection
我是不是省略太多了，以后还是要把话说清楚

---

还真是，我理解能力太差了